src/GafferImage/ImageWriter.cpp

```diff
@@ -1875,6 +1887,11 @@ void ImageWriter::execute() const
 
 	std::string constantColorSpaceStr;
 
+	const ValuePlug *optionsPlug = this->getChild<ValuePlug>( out->format_name() );
+	const StringPlug *dataTypePlug = optionsPlug ? optionsPlug->getChild<StringPlug>( g_dataTypePlugName ) : nullptr;
+	const StringPlug *depthDataTypePlug = optionsPlug ? optionsPlug->getChild<StringPlug>( g_depthDataTypePlugName ) : nullptr;
+	bool overrideDepthToFloat = depthDataTypePlug ? depthDataTypePlug->getValue() == "float" : false;
```

Comment:
Is there any reason we store `bool overrideDepthToFloat` rather than `string depthDataTypeOverride`? It seems like the latter would be a tiny bit simpler, and would cope with other data types (in theory).